native/package.json

```diff
@@ -75,7 +76,7 @@
     "react-native-permissions": "2.0.9",
     "react-native-progress": "^4.0.3",
     "react-native-reanimated": "^1.13.2",
-    "react-native-render-html": "^5.0.0-alpha.2",
+    "react-native-render-html": "^5.1.0",
```

Comment:
@Taggotty does this version already fix the issues with RTL/LTR?

Reply:
No, the issue is still open: https://github.com/meliorence/react-native-render-html/issues/286